src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -0,0 +1,11 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class ColorSupplier {
+    public String getRandomColor() {
+        Random random = new Random();
+        Color[] color = Color.values();
+        return color[random.nextInt(color.length)].name().toLowerCase();
+    }
```

Comment:
Random should be a class variable
```suggestion
    public String getRandomColor() {
        return Color.values()[random.nextInt(Color.values().length)].name().toLowerCase();
    }
```